build/zip.py

```diff
@@ -99,11 +99,19 @@ def main(argv):
             if basename == 'chrome_sandbox'
             else dep
           )
+          name_to_write = arcname
+          if should_flatten:
+            if flatten_relative_to:
+              print()
```

Comment:
I think I left this here by accident

Reply:
Lol, my fault for not catching it though! Will remove, thanks 🙂